### DISCUSSION TITLE
How to implement a multiple select?

### DISCUSSION THREAD

**Discussion by ricardo-valero** [pablo-abc/felte#103] [Q&A] [answered]
I was working with a simple `solid` project where I needed a multiple select, how would you do this with `felte`? 

I forked the example and changed the type select field with the `multiple` attribute 
https://codesandbox.io/s/felte-v1-demo-solidjs-forked-ocm4qh?file=/src/main.tsx

**Comment by pablo-abc** [comment]
I'll add an example soon! There seems to have been a regression as well and this is not working currently. I'll create an issue from this.

**Comment by pablo-abc** [comment, marked as answer]
Your example should work without any changes as of `@felte/solid@1.0.2`. A slight difference from how Svelte handles it is that we take the values as the DOM would have theme in the case of `<select>` elements. That means that the value you'll receive will always be a string or an array of strings in the case of `<select multiple>`. 

Nothing prevents you from assigning a stringified JSON as value, you'd need to parse it on submission or transform it using a `transform` function.
